README.md

```diff
@@ -1 +1,143 @@
-# math-magician1
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+PROJECT DESCRIPTION -->
+
+# 📖 [math-magician]
+
+> Describe your project in 1 or 2 sentences.
+
+**[math-magician]** "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+    Make simple calculations.
+    Read a random math-related quote.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>FrontEnd</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li><a href="https://jestjs.io/docs/tutorial-react">jest</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features 
+
+A react logo
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo 
+
+- [Live Demo Link](coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started 
+
+To get a local copy up and running follow these simple example steps.
+## Prerequisites
+
+  1. You need to have Code Editor Installed
+  2. Make sure Nodejs and NPM is installed in your system for linters and libraries
+
+## Setup
+
+1. Clone the repo in your local machine:
+
+```bash
+git clone git clone https://github.com/khanetmoi/math-magician-app
+```
+2. Open terminal and cd into project and install the dependencies
+```bash
+cd Math-Magician
+npm install
+```
+
+3. run the command to open your index.html automatically in your brower:
+```bash
+npm run build
+npm start
+```
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```

Comment:
- [ ] Kindly consider removing this line as this text is directed to the author of this repo and not to the visitor. removing this text will make your repo more professional.